web/app_options.js

```diff
@@ -179,6 +179,16 @@ const defaultOptions = {
     value: 0,
     kind: OptionKind.VIEWER + OptionKind.PREFERENCE,
   },
+  pageForegroundColor: {
+    /** @type {string} */
+    value: "CanvasText",
+    kind: OptionKind.VIEWER + OptionKind.PREFERENCE,
+  },
+  pageBackgroundColor: {
+    /** @type {string} */
+    value: "Canvas",
+    kind: OptionKind.VIEWER + OptionKind.PREFERENCE,
+  },
```

Comment:
Please note that all of the options are sorted *alphabetically*, within each OptionKind-section, in this file.
Hence, please flip the order of these options and also move them up to their "correct" spot.